src/General/Button/PrimaryButton.tsx

```diff
@@ -38,6 +40,7 @@ interface Props extends React.ComponentPropsWithoutRef<typeof PrimaryBtn> {
   block?: boolean;
   small?: boolean;
   removeHoverEffect?: boolean;
+  tag?: React.ElementType;
```

Comment:
On an unrelated note, we should really abstract all these repeated interfaces to a common interface :stuck_out_tongue: Don't have to do it in this PR though.

Reply:
That's true! Well, I guess the storybook is still changing, somebody might improve this eventually ^^